bench/commands/utils.py

```diff
@@ -116,8 +116,9 @@ def backup_all_sites():
 @click.option('--develop', default='develop')
 @click.option('--master', default='master')
 @click.option('--owner', default='frappe')
-def release(app, bump_type, develop, master, owner):
+@click.option('--repo-name')
+def release(app, bump_type, develop, master, owner, repo_name):
 	"Release app (internal to the Frappe team)"
 	from bench.release import release
-	repo = os.path.join('apps', app)
+	repo = os.path.join('apps', repo_name) if repo_name else os.path.join('apps', app)
```

Comment:
No. It should be used just before pushing the release!

Basically, the folder name can be different from the repo name, and when pushing the release, you have to use the github repo name and not the folder name if it is different. See the line where I commented.